core/src/main/java/com/amplitude/core/utilities/Response.kt

```diff
@@ -129,33 +129,33 @@ class FailedResponse(response: JSONObject) : Response {
 }
 
 interface ResponseHandler {
-    fun handle(response: Response) {
+    fun handle(response: Response, events: Any, eventsString: String) {
```

Comment:
@qingzhuozhen can you explain why the change to move events/eventString to ResponseHandlers from Storage?

Reply:
The reason for this is just to track the retry times in Response handler.